pages/blog/posts/6-river-systems-case-study.md

```diff
@@ -113,4 +113,4 @@ While 6 River Systems had previously used JSON Schema in a number of areas acros
 
 Thank you to [Jacob Thomas](https://www.linkedin.com/in/jake-thomas/) and 6 River Systems for enabling us to share this case study with you.
 
-Images used with permission.
\ No newline at end of file
+Images used with permission.
```

Comment:
@DarhkVoyd this is an unnecessary change right??

Reply:
Yeah, that's work of my markdown linter. Not sure if it changes anything.